dockertest/docker_daemon.py

```diff
@@ -126,11 +126,17 @@ def which_docker():
     possible, but unsupported, so we're not going to worry about it.
     """
     docker = 'docker'
-    with open('/etc/sysconfig/docker', 'r') as docker_sysconfig:
-        for line in docker_sysconfig:
-            if line.startswith('DOCKERBINARY='):
-                if 'docker-latest' in line:
-                    docker = 'docker-latest'
+
+    # /etc/sysconfig/docker doesn't exist on the CI host on which
+    # we run github unit tests
```

Comment:
Hrmmm, or it's not called docker, or it's not installed, or ... lots of ways this could break 😢 

This feels wrong to me: Changing behavior in code when it _guesses_ it's running under test.  What happens when it guesses wrong...it silently fails.  VW had a similar problem with their TDIs and the EPA, maybe a bad analogy, but same idea.